tests/test_ldd.py

```diff
@@ -0,0 +1,250 @@
+                found=True,
+                mem_address="",
+                unused=False),
+            lddwrap.Dependency(
+                soname="libdl.so.2",
+                path=pathlib.Path("/lib/x86_64-linux-gnu/libdl.so.2"),
+                found=True,
+                mem_address="",
+                unused=False),
+            lddwrap.Dependency(
+                soname=None, path=pathlib.Path("/lib64/ld-linux-x86-64.so.2"), found=True, mem_address="",
+                unused=False),
+            lddwrap.Dependency(
+                soname="libpthread.so.0",
+                path=pathlib.Path("/lib/x86_64-linux-gnu/libpthread.so.0"),
+                found=True,
+                mem_address="",
+                unused=False)
+        ]
+
+        for i, dep in enumerate(deps):
+            self.assertTrue(
+                equals(dep, expected_deps[i]), "Incorrect dependency, expected {}\ngot {}".format(
+                    expected_deps[i], dep))
+            self.assertEqual(dep.unused, expected_deps[i].unused, "Incorrect unused state in dependency: {}".format(
+                dep.path))
+
+    def test_dir_verbosed_and_unused(self):
+        path = pathlib.Path("/bin/dir")
+        deps = lddwrap.get_dependencies(path=path, verbose=True, unused=True)
+
+        expected_deps = [
+            lddwrap.Dependency(soname="linux-vdso.so.1", path=None, found=True, mem_address=""),
+            lddwrap.Dependency(
+                soname="libselinux.so.1",
+                path=pathlib.Path("/lib/x86_64-linux-gnu/libselinux.so.1"),
+                found=True,
+                mem_address="",
+                unused=True),
+            lddwrap.Dependency(
+                soname="libc.so.6", path=pathlib.Path("/lib/x86_64-linux-gnu/libc.so.6"), found=True, mem_address=""),
+            lddwrap.Dependency(
+                soname="libpcre.so.3",
+                path=pathlib.Path("/lib/x86_64-linux-gnu/libpcre.so.3"),
+                found=True,
+                mem_address=""),
+            lddwrap.Dependency(
+                soname="libdl.so.2", path=pathlib.Path("/lib/x86_64-linux-gnu/libdl.so.2"), found=True, mem_address=""),
+            lddwrap.Dependency(
+                soname=None, path=pathlib.Path("/lib64/ld-linux-x86-64.so.2"), found=True, mem_address=""),
+            lddwrap.Dependency(
+                soname="libpthread.so.0",
+                path=pathlib.Path("/lib/x86_64-linux-gnu/libpthread.so.0"),
+                found=True,
+                mem_address="")
+        ]
+
+        for i, dep in enumerate(deps):
+            self.assertTrue(
+                equals(dep, expected_deps[i]), "Incorrect dependency, expected {}\ngot {}".format(
+                    expected_deps[i], dep))
+
+    def test_version(self):
+        version = lddwrap.version()
+        self.assertEqual(type(version[0]), int)
+        self.assertEqual(type(version[1]), int)
+
+        version = str(version[0]) + "." + str(version[1])
```

Comment:
This variable name was used before. It can be confusing to read. How about `version_str`?